blob-clients/blob-client-jersey2/src/main/java/com/bazaarvoice/emodb/blob/client/BlobStoreClientFactory.java

```diff
@@ -1,24 +1,24 @@
-package com.bazaarvoice.emodb.blob.jersey2.client;
+package com.bazaarvoice.emodb.blob.client;
 
-import com.bazaarvoice.emodb.common.jersey2.Jersey2EmoClient;
+import com.bazaarvoice.emodb.client.Jersey2EmoClient;
 import org.glassfish.jersey.client.ClientProperties;
 
 import javax.ws.rs.client.Client;
 import java.net.URI;
 
 
-public class BlobStoreJersey2ClientFactory extends AbstractBlobStoreJersey2ClientFactory {
+public class BlobStoreClientFactory extends AbstractBlobStoreClientFactory {
```

Comment:
Do we really need to make name change for all added newer client files/packages? As example, the above change would result in two clientfactory files of jersey1 and jersey 2 with same naming. Based on my experience while upgrading to newer clients, even with different artifact-ids, it just increases the complexity for debugging due to same package and file names.

Reply:
i would use the same files/packages so it will be easier to upgrade (and upgrade runbook steps will be reduced). from my experience, haven't had any issues with client upgrades (maven-dependency-plugin and maven-enforcer-plugin really help).